spec/controllers/posts_controllers_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
+  describe 'GET #index' do
+    it 'returns http success' do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET #show' do
```

Comment:
* Kindly Create a Request spec file for all your controllers as stated in the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Create%20a%20Request%20spec%20file%20for%20all%20your%20controllers.)